frontend/src/shared/ui/Button/Button.stories.ts

```diff
@@ -10,7 +13,11 @@ const meta: Meta<typeof Button> = {
 		onClick: { action: "clicked" },
 		width: { control: "text" },
 		height: { control: "text" },
-		primary: { control: "boolean" },
+		variant: {
+			control: "select",
+			options: ["primary", "secondary", "gray"] as ButtonVariant[],
```

Comment:
Дважды переопределяешь один и тот же тип
```suggestion
			options: ["primary", "secondary", "gray"] as ButtonVariant[],
```